File: app/src/main/java/movies/test/softserve/movies/viewmodel/FragmentViewModel.java
package movies.test.softserve.movies.viewmodel;

import android.arch.lifecycle.ViewModel;

import movies.test.softserve.movies.fragment.MovieFragment;
import movies.test.softserve.movies.fragment.WatchedFragment;

/**
 * Created by rkrit on 01.11.17.
 */

public class FragmentViewModel extends ViewModel {
    private MovieFragment movieFragment;
    private WatchedFragment watchedFragment;


    public MovieFragment getMovieFragment() {
        return movieFragment;
    }

    public void setMovieFragment(MovieFragment movieFragment) {
        this.movieFragment = movieFragment;
    }

    public WatchedFragment getWatchedFragment() {
        return watchedFragment;
    }

    public void setWatchedFragment(WatchedFragment watchedFragment) {
        this.watchedFragment = watchedFragment;
    }
}

File: app/src/main/java/movies/test/softserve/movies/viewmodel/FragmentViewModel.kt
package movies.test.softserve.movies.viewmodel

import android.arch.lifecycle.ViewModel
import movies.test.softserve.movies.PersonalCabinetFragment
import movies.test.softserve.movies.fragment.*

/**
 * Created by rkrit on 02.11.17.
 */
class FragmentViewModel : ViewModel() {
    var movieFragment: MovieFragment? = null
    var watchedFragment: WatchedFragment? = null
    var genresFragment: GenreFragment? = null
    var tvShowFragment: TVShowFragment? = null
    var searchFragment: SearchFragment? = null
    var achievementsFragment: AchievementsFragment? = null
    var personalCabinetFragment: PersonalCabinetFragment? = null
}